packages/launcher/lib/utils.ts

```diff
@@ -1,12 +1,33 @@
 import execa from 'execa'
 import cp from 'child_process'
+import os from 'os'
 import Bluebird from 'bluebird'
 
 // export an object for easy method stubbing
 export const utils = {
   execa,
+  spawnWithArch: <T extends cp.SpawnOptionsWithStdioTuple<cp.StdioNull, cp.StdioPipe, cp.StdioPipe>>(cmd: string, args: string[], opts: T) => {
+    if (!(os.platform() === 'darwin' && os.arch() === 'arm64')) {
```

Comment:
can you add a comment here...

"If we're not on OSX or if the binary was not built in arm64 then we can safely spawn the browser which will get launched as x86_64"

Reply:
I also might change the logic to be this...

```js
if ((os.platform() !== 'darwin' || os.arch() !== 'arm64')) {
```

that'll make the condition be affirmative instead of negative affirmative

Reply:
3d1badc4b6fe422b0e60d07f743a24d69b6aed78